arbnode/node.go

```diff
@@ -410,7 +429,8 @@ func createNodeImpl(
 	}
 
 	transactionStreamerConfigFetcher := func() *TransactionStreamerConfig { return &configFetcher.Get().TransactionStreamer }
-	txStreamer, err := NewTransactionStreamer(arbDb, l2Config, exec, broadcastServer, fatalErrChan, transactionStreamerConfigFetcher)
+	snapSyncConfigFetcher := func() *SnapSyncConfig { return &configFetcher.Get().SnapSyncTest }
+	txStreamer, err := NewTransactionStreamer(arbDb, l2Config, exec, broadcastServer, fatalErrChan, transactionStreamerConfigFetcher, snapSyncConfigFetcher)
```

Comment:
since this isn't a dynamic config, no need to pass a fetcher.
you can get once: configFetcher.Get().SnapSyncTest and pass that to NewTransactionStreamer. It cannot change so don't need a fecher for it.

Reply:
Done.